app/Templates.hs

```diff
@@ -1,12 +1,31 @@
+{-# LANGUAGE DeriveGeneric #-}
 {-# LANGUAGE TemplateHaskell #-}
-
+{-# LANGUAGE OverloadedStrings #-}
 module Templates where
 
+import GHC.Generics
 import Data.FileEmbed
 import System.FilePath ((</>))
+import qualified Data.Text.Lazy as LT
+import qualified Data.Text.Lazy.Encoding as LT
+import qualified Data.Text as T
+import Data.Aeson
+import Data.Aeson.TH
+import Data.Aeson.Encode.Pretty
+
+data PscPackage = PscPackage { name :: T.Text
```

Comment:
cant we use normal formatting here instead of weird column align

Reply:
this should also probbaly go in a Types module since we will probably end up needing thsi later